plugins/experimental/volcano/fetch_policy.h

```diff
@@ -0,0 +1,69 @@
+/*
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+/**
+ * @file volcano_policy.h
+ * @brief Fetch policy interface (header file).
+ */
+
+#ifndef PLUGINS_EXPERIMENTAL_VOLCANO_POLICY_H_
+#define PLUGINS_EXPERIMENTAL_VOLCANO_POLICY_H_
+
+#include <list>
+#include <openssl/sha.h>
+#include <string.h>
+#include <string>
+#include <unordered_map>
+
+#include "common.h"
+
+class FetchPolicy;
+class SimplePolicy;
+class Prescaler;
+
+/**
+ * @brief Fetch policy interface.                                                    *
```

Comment:
Remove '*' at the end

Reply:
@maskit, I would prefer to leave the `*`, it is a [Doxygen](https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html) style comment.

Reply:
Is it ??? I'm talking about this `*`.

![image](https://user-images.githubusercontent.com/153144/42354312-e700ab56-8100-11e8-81db-1e276f57df77.png)